tests/test_scripts.py

```diff
@@ -54,6 +55,21 @@ def test_load_into_counting():
     assert os.path.exists(outfile)
 
 
+def test_load_into_counting_nonwritable():
+    script = scriptpath('load-into-counting.py')
+    args = ['-x', '1e3', '-N', '2', '-k', '20', '-t']
+
+    outfile = utils.get_test_data('test-nonwritable')
+    os.chmod(outfile, stat.S_IWOTH | stat.S_IRUSR)
```

Comment:
You might as well create a new file using `tempfile.mkdtemp` and setting the permissions bits on it. No need to include it in the `test-data` directory if it isn't going to have the permissions it advertises. Though git does allow you to check files in with 'a-w' perms.